scripts/go-to-slide.js

```diff
@@ -7,8 +7,10 @@ H5P.CoursePresentation.GoToSlide = (function ($) {
    * @class
    * @param {Number} slideNum
    * @param {CoursePresentation} cp
+   * @param {String} goToSlideNextPrevious
+   * @param {Number} index
    */
-  function GoToSlide(title, slideNum, invisible, cp) {
+  function GoToSlide(title, slideNum, invisible, cp, goToSlideNextPrevious, index) {
```

Comment:
I think index would be cp.currentSlideIndex. Could you try using that instead of index? At least we would get rid of that parameter.  Maybe nextPreviousSlide could be renamed to goToSlideType or something now that it has a third option? Must also make sure that these changes work for old content. Give the new option a default in the CP "constructor" function.